cfme/common/provider_views.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+from widgetastic.widget import View, Text
+
+from cfme import BaseLoggedInPage
+from widgetastic_patternfly import Dropdown
+from widgetastic_manageiq import DetailsToolBarViewSelector
+from widgetastic_manageiq import (BreadCrumb,
+                                  SummaryTable,
+                                  Button,
+                                  TimelinesView)
+from utils.version import current_version
```

Comment:
Are we okay adding more utils.version.current_version uses when we're also actively trying to remove it?